gps_common/src/utm_odometry_node.cpp

```diff
@@ -3,245 +3,170 @@
  */
 
 #include <ros/ros.h>
-#include <message_filters/subscriber.h>
-#include <message_filters/time_synchronizer.h>
-#include <sensor_msgs/NavSatStatus.h>
-#include <sensor_msgs/NavSatFix.h>
 #include <gps_common/conversions.h>
 #include <nav_msgs/Odometry.h>
 #include <novatel_gps_msgs/Inspva.h>
-#include <geometry_msgs/TransformStamped.h>
-#include <geometry_msgs/QuaternionStamped.h>
 #include <tf/transform_broadcaster.h>
-#include <tf/transform_listener.h>
 #include <geometry_msgs/PoseStamped.h>
 #include <math.h>
+#include <Eigen/Dense>
 
 using namespace gps_common;
 
-ros::Publisher odom_pub, world_odom_pub;
-ros::Subscriber gps_source_sub;
-std::string frame_id, child_frame_id, world_frame_id, world_child_frame_id;
-double rot_cov, y_offset, x_offset, z_offset;
-double roll_offset, pitch_offset, yaw_offset;
-bool odom_init, publish_odom_tf, broadcast_world_tf;
-
-
-void callback(const sensor_msgs::NavSatFixConstPtr& fix) {
-  if (fix->status.status == sensor_msgs::NavSatStatus::STATUS_NO_FIX) {
-    ROS_DEBUG_THROTTLE(60,"No fix.");
-    return;
-  }
+static double radians_to_degrees = 180.0 / M_PI;
+static double degrees_to_radians = M_PI / 180.0;
```

Comment:
maybe it should be a constant.
Also the variable name makes me think that is does something
 why not making a function or a more c style define It would be prettier to read in code and more logic imho.

Reply:
- [ ] Make these a "#define".